www/view-single.html

```diff
@@ -12,6 +12,12 @@ <h1 class="text-center text-lg-left">{{ stream.name }} <a id="btn-geo-view" clas
           </div>
           {% endfor %}
         </div>
+        <div class="bootstrap-tagsinput">
+          {% for tag in stream.tags %}
+          <span class="tag label label-info">{{ tag }}</span>
```

Comment:
I'll take a look at the styling. Worse case scenario we keep it for debugging and css hide it.